src/loader.ts

```diff
@@ -544,7 +544,7 @@ declare const Packages: {} | undefined;
 	}
 
 	function getModuleInformation(moduleId: string, referenceModule?: DojoLoader.Module): DojoLoader.Module {
-		let match = moduleId.match(/^([^\/]+)(\/(.+))?$/);
+		let match = moduleId.match(/^((?:@[^\/]*\/)?[^\/]+)(\/(.+))?$/);
```

Comment:
I know you were fixing this, but I think we should clean this regex up.

It seems like the second capture group [isn't used](https://regexper.com/#%2F%5E((%3F%3A%40%5B%5E%5C%2F%5D*%5C%2F)%3F%5B%5E%5C%2F%5D%2B)(%5C%2F(.%2B))%3F%24%2F) and should be `?:` out.

<img width="695" alt="regexper" src="https://user-images.githubusercontent.com/1282577/28819600-aa900d7a-76a6-11e7-850c-a0583ad45ee4.png">

Also, we should use destructuring to be more _modern_ in the way we deal with this.